### ISSUE TITLE
'ID2D1RenderTarget' is not defined on macOS

### ISSUE THREAD

**Issue by AlecHaring** [sciter-sdk/pysciter#43]
There seems to be an issue with 0.5.0 on macOS. I get this error:
```
Traceback (most recent call last):
  File "/Users/alecharing/Desktop/Python Projects/sciter_testing/test.py", line 2, in <module>
    import sciter
  File "/Users/alecharing/Desktop/Python Projects/sciter_testing/venv/lib/python3.8/site-packages/sciter/__init__.py", line 18, in <module>
    from .capi.scapi import SciterAPI
  File "/Users/alecharing/Desktop/Python Projects/sciter_testing/venv/lib/python3.8/site-packages/sciter/capi/scapi.py", line 46, in <module>
    SciterRenderD2D = SCFN(BOOL, HWINDOW, POINTER(ID2D1RenderTarget))
NameError: name 'ID2D1RenderTarget' is not defined
```
which originates in the scapi.py file. Looking at the git revision history, it seems like `ID2D1RenderTarget` used to only be referenced if `SCITER_WIN` was true. Should this still be the case?

**Comment by pravic** [comment]
Could you check 27a43ab153?

**Comment by AlecHaring** [comment]
Works like a charm! Thank you

**Comment by pravic** [comment]
Published as 0.5.1
